### PR TITLE
Auth settings

### PR DESCRIPTION
**Pull request by f1401martin**
@bp1222 you had some thoughts on this, feel free to make changes...

### CODE REVIEW

**Review by bp1222 — APPROVED** [review]

**Review by mattdodge — APPROVED** [review]

**Comment by mattdodge** [comment]
Will merge this once PK Auth goes live

**Comment by aaronranard** [comment]
@f1401martin @mattdodge bump. PK Auth is live

**Comment by mattdodge** [comment]
Awesome, @f1401martin are these the right settings? Now that we have the pk server standalone conf file they should go there, not in nio.conf anymore. But I want to make sure these settings are correct, they will need to be read by the module in order to work, the pk server won't read from this conf file I don't think

**Comment by f1401martin** [comment]
These are the right settings if nio were to read them (they get somewhat transformed to what a pubkeeper server understand,) but you bring an interesting point, if nio would not start a pubkeeper server on its own any longer, then none of this is needed, including the code in the module that starts it

**Comment by mattdodge** [comment]
nio will still start a pubkeeper server, it's just that the `pubkeeper.server.core` library won't be included in the nio binary wheel anymore. So if these settings are read by the pk module and relayed to the embedded pk server then we're good to go. I'll move them over to the pk server conf partial and test it out, then push up a new commit here.

File: nio.conf [hunk 1]
```diff
@@ -267,6 +264,28 @@ port=[[PKS_PORT]]
...
+
+# Below are example settings using LocalAuth.  Modules and Providers
+# will be provided this directive for their own configuration
+
+# validate is the URL used by LocalAuthProvider to lookup the provided token
+# for local environments set PKS_AUTH_VALIDATE_URL=http://localhost:8002/validate
+#auth_validate = [[PKS_AUTH_VALIDATE_URL]]
+
+# IP Address of the Auth Server
+#auth_ip_address      = 0.0.0.0
```
Comment (bp1222) [inline nio.conf]: This is no longer needed.

File: nio.conf [hunk 2]
```diff
@@ -267,6 +264,28 @@ port=[[PKS_PORT]]
...
+
+# Port of the Auth Server
+#auth_port            = 8001
```
Comment (bp1222) [inline nio.conf]: This is no longer needed

**Review by bp1222 — CHANGES_REQUESTED** [review]
Legacy fields that are no longer needed.

File: nio.conf (hunk 1 above)
Comment (mattdodge) [reply nio.conf]: Yeah, I'm reorganizing this, stand by for a push. It will affect the pk module code too

**Comment by mattdodge** [comment]
Closing in favor of #69, which has a more up to date history

**Pull request closed without merging**